Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -761,7 +761,7 @@ def _on_resume_err(self, failure):
         failure.trap(CancelledError)
         self._logger.error("Resume data cancelled")
 
-    @waitForHandleAndSynchronize()
+    @checkHandleAndSynchronize(default=Deferred().callback(None))
```

Comment:
I haven't looked at anything else than this particular line but read abut `defer.succeed()` instead of pre-firing a `Deferred`